core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -435,15 +443,13 @@ class KRaftClusterTest {
 
   private def doOnStartedKafkaCluster(numControllerNodes: Int = 1,
                                       numBrokerNodes: Int,
-                                      brokerPropertyOverrides: (TestKitNodes, BrokerNode) => Map[String, String])
+                                      brokerPropertyOverrides: util.Map[Integer, util.Map[String, String]])
                                      (action: KafkaClusterTestKit => Unit): Unit = {
     val nodes = new TestKitNodes.Builder()
```

Comment:
Could we change the style to `TestKitNodes.builder()` so as to unify the builder pattern in testing?